services/vqa-service/service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+  
+service VqaService {
+    rpc answer(VqaRequest) returns (VqaResponse);
+}
+
+message VqaRequest {
+    string question = 1;
+    string image_format = 2;
+    bool use_pm = 3;
+    bytes data = 4;
+}
+
+message VqaResponse {
+    string message = 1;
```

Comment:
It is not clear what does `message` mean here, especially taking into account `error_message` is here as well. Should we name it `answer`?